public/app/components/panelclipboard.js

```diff
@@ -0,0 +1,31 @@
+define([
+  'lodash',
+  'store'
+],
+function(_, store) {
+  'use strict';
+
+  return {
+    get: function() {
+      var json = store.get("copiedPanel");
+      var panel = null;
+      try {
+        panel = JSON.parse(json);
+      } catch(err) {
+        this.clear();
+      }
+      return panel;
+    },
+    set: function(panel) {
+      var copy = _.clone(panel);
```

Comment:
Thanks for the heads-up! I've changed it to what you suggested, and yes - it does seem to delete the &&hashKey automatically.